CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `log2` and `log` math functions were adjusted for consistency in error throwing: PR [#342](https://github.com/tact-lang/tact/pull/342)
 - Shadowing built-in static functions is now forbidden: PR [#351](https://github.com/tact-lang/tact/pull/351)
+- All integer and boolean expressions are now being attempted to be evaluated as constants. Additionally, compile-time errors are thrown for errors encountered during the evaluation of actual constants: PR [#352](https://github.com/tact-lang/tact/pull/352)
```

Comment:
Do we const-evaluate things like `const x: Int = 42 ? 1 : 0`? If this is not the case, we should say something like `More ...` instead of `All ...`

Reply:
such expressions are being attempted to be const-evaluated, but due to incompleteness of the const-evaluation itself it doesn't succeed.